config/params.ini

```diff
@@ -1,6 +1,6 @@
 ; There is detailed documentation for each parameter in the README.md
 [general]
-name=Botty
+name=Botty Barbarian
```

Comment:
Please change only change the default values of already existing params if really needed.